src/pycram/process_modules/hsrb_process_modules.py

```diff
@@ -52,65 +33,31 @@ def _execute(self, desig: MoveMotion):
         robot.set_pose(desig.target)
 
 
-class HSRBMoveHead(ProcessModule):
-    """
-    This process module moves the head to look at a specific point in the world coordinate frame.
-    This point can either be a position or an object.
-    """
-
-    def _execute(self, desig: LookingMotion):
-        target = desig.target
-        robot = World.robot
-
-        local_transformer = LocalTransformer()
-        pose_in_pan = local_transformer.transform_pose(target, robot.get_link_tf_frame("head_pan_link"))
-        pose_in_tilt = local_transformer.transform_pose(target, robot.get_link_tf_frame("head_tilt_link"))
-
-        new_pan = np.arctan2(pose_in_pan.position.y, pose_in_pan.position.x)
-        new_tilt = np.arctan2(pose_in_tilt.position.z, pose_in_tilt.position.x ** 2 + pose_in_tilt.position.y ** 2) * -1
-
-        current_pan = robot.get_joint_position("head_pan_joint")
-        current_tilt = robot.get_joint_position("head_tilt_joint")
-
-        robot.set_joint_position("head_pan_joint", new_pan + current_pan)
-        robot.set_joint_position("head_tilt_joint", new_tilt + current_tilt)
-
-
-class HSRBMoveGripper(ProcessModule):
-    """
-    This process module controls the gripper of the robot. They can either be opened or closed.
-    Furthermore, it can only moved one gripper at a time.
-    """
-
-    def _execute(self, desig: MoveGripperMotion):
-        robot = World.robot
-        gripper = desig.gripper
-        motion = desig.motion
-        for joint, state in RobotDescription.current_robot_description.get_arm_chain(gripper).get_static_gripper_state(motion).items():
-            robot.set_joint_position(joint, state)
-
-
 class HSRBDetecting(ProcessModule):
     """
     This process module tries to detect an object with the given type. To be detected the object has to be in
     the field of view of the robot.
     """
-
-    def _execute(self, desig: DetectingMotion):
-        rospy.loginfo("Detecting technique: {}".format(desig.technique))
-        robot = World.robot
-        object_type = desig.object_type
-        # Should be "wide_stereo_optical_frame"
-        cam_frame_name = RobotDescription.current_robot_description.get_camera_frame()
-        # should be [0, 0, 1]
-        front_facing_axis = RobotDescription.current_robot_description.get_default_camera().front_facing_axis
+   # pass
+    def _execute(self, desig: DetectingMotion) -> Any:
+        pass
+        # rospy.loginfo("Detecting technique: {}".format(desig.technique))
+        # robot = World.robot
```

Comment:
Large blocks of commented code are not needed since we have VCS

Reply:
what do u mean by that? This code cannot work without the other PR therefore i commented it out for the time being, do u want me to delete it?